### ISSUE TITLE
Dataset spilt problem

### ISSUE THREAD

**Issue by counterattack-hub** [ucla-mobility/V2V4Real#27]
Hi, thanks for your excellent work！

However, I have one problem that I can't understand how to spilt the dataset in your shared link.

<img width="504" alt="微信图片_20240425220356" src="https://github.com/ucla-mobility/V2V4Real/assets/85003855/d0a283b3-5078-45be-8319-90092147759e">

In this three folders, which one is training set, which one is validation set, and which one is test set. How do I organize them into the project.

**Comment by marcoCalipari** [comment]
Hi, could you please provide an answer to this?
